### PR TITLE
Fix (Enroll now) for University Courses

### PR DESCRIPTION
**Pull request by shadinaif**
Fix (Enroll now) for University Courses

-------------------------
Related to https://edraak.atlassian.net/browse/OU-287

### CODE REVIEW

File: lms/djangoapps/edraak_university/mixins.py [hunk 1]
```diff
@@ -48,4 +48,5 @@ def get_context_data(self, **kwargs):
         data = super(ContextMixin, self).get_context_data(**kwargs)
         data['course'] = self.get_course()
         data['is_staff'] = self.is_staff()
+        data['course_key'] = self.get_course_key()
```
Comment (ghost) [inline lms/djangoapps/edraak_university/mixins.py:51]: Add a unit test for this one please.

File: lms/djangoapps/edraak_university/views.py [hunk 2]
```diff
@@ -57,6 +58,8 @@ def get(self, *args, **kwargs):
         if self.is_staff():
             return redirect('edraak_university:id_staff', course_id=course.id)
 
+        CourseTabView.register_user_access_warning_messages(self.request, self.get_course_key())
```
Comment (ghost) [inline lms/djangoapps/edraak_university/views.py:61]: Also this one. Not super complicated, but something basic to ensure it's not lost during future merges.

**Review by ghost — APPROVED** [review]
Other the than tests, it looks good to me. Thanks Shadi!

File: lms/djangoapps/edraak_university/views.py (hunk 2 above)
Comment (shadinaif) [reply lms/djangoapps/edraak_university/views.py:61]: Done :+1:

**Pull request merged**